sky/data/storage_utils.py

```diff
@@ -162,3 +197,12 @@ def get_excluded_files_from_gitignore(src_dir_path: str) -> List[str]:
                     to_be_excluded += '*'
                 excluded_list.append(to_be_excluded)
     return excluded_list
+
+
+def get_excluded_files(src_dir_path: str) -> List[str]:
+    """ List files and directories to be excluded.
+    """
+    excluded_list: List[str] = []
+    excluded_list.extend(get_excluded_files_from_skyignore(src_dir_path))
+    excluded_list.extend(get_excluded_files_from_gitignore(src_dir_path))
```

Comment:
If a file is included in both skyignore and gitignore, will it get added twice to the excluded list? If so, should we make excluded_list a set?

Reply:
I think we should just only use `.skyignore` for ignoring files uploading to sky clusters without merging with `.gitignore` to make things super clear without making more of these "implicit" rules about how to resolve conflicts between the two files. I don't think explicitly copying certain items in `.gitignore` to `.skyignore` is that much work for users, but def will make things clear and straightforward.

Reply:
Yes, that is what I meant, but it seems that these two lines are merging the two files?